setup.py

```diff
@@ -129,7 +133,7 @@ def get_versions(default={}, verbose=False):
     packages=find_packages('src', exclude=['leap.keymanager.tests']),
     package_dir={'': 'src'},
     test_suite='leap.keymanager.tests',
-    install_requires=utils.parse_requirements(),
```

Comment:
you could use `utils.parse_requirements(reqfiles=["requirements.pip", "pkg/requirements-leap.pip"])`